src/Widgets/VerticalNavigationWidget/VerticalNavigationWidgetComponent.tsx

```diff
@@ -84,40 +84,32 @@ const SubNavItems = connect(
 
 const ExpandableNavItem = connect(
```

Comment:
This is no longer "expandable". I suggest to rename it to `NestedNavItem`. I would also change the `key` to `VerticalNavigationWidget-nested-${obj.id()}`.